src/util/util.js

```diff
@@ -16,7 +16,8 @@ export function required() {
 
 const GRAPHIQL_VERSION = '0.7.1';
 
-export function renderGraphiQL({ query, variables, version = GRAPHIQL_VERSION } = {}) {
+export function renderGraphiQL({ query, variables, version = GRAPHIQL_VERSION, csrfToken } = {}) {
+  csrfToken = csrfToken ? `,'x-csrf-token': '${csrfToken}'` : '';
```

Comment:
It's usually a good idea to treat function parameters as immutable. In this case a better name could perhaps be `csrfHeader`, as it's more aligned with the actual value.